src/app/components/MediaLoader/index.test.tsx

```diff
@@ -30,6 +32,49 @@ describe('MediaLoader', () => {
     (useState as jest.Mock).mockImplementation(() => [false, () => false]);
   });
 
+  describe('Portrait title', () => {
+    it('Is rendered when video is portrait, pageType is not mediaArticle, and embedded prop is false', () => {
+      render(
+        <MediaPlayer
+          blocks={aresMediaPortraitBlocks as MediaBlock[]}
+          embedded={false}
+        />,
+        {
+          id: 'testId',
+          pageType: TV_PAGE,
```

Comment:
Let's use ARTICLE_PAGE instead, as currently it's the only other page which supports Portrait Video